doc/source/train/examples.yml

```diff
@@ -54,6 +54,26 @@ examples:
       - natural language processing
     contributor: community
     link: examples/intel_gaudi/bert
+  - title: "Fine-tune Llama-2-7b with Intel Gaudi"
+    frameworks:
+      - accelerate
+      - transformers
+    skill_level: intermediate
+    use_cases:
+      - natural language processing
+      - large language models
+    contributor: community
+    link: examples/intel_gaudi/llama
```

Comment:
Can we move this to the end of the intermediate section in the yaml file, similar to the ordering of the beginner skill level sectoin?

I think it is better for the examples in each "skill level" to start with more familiar usage, then get more advanced into using custom accelerators.

Reply:
Updated!